src/Chapter1.hs

```diff
@@ -556,7 +553,8 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+isVowel :: Char -> Bool
+isVowel = (elem ['a','e','i','o','u'])
```

Comment:
Absolutely correct! And you use eta-reduction 👍🏼 

One note in here, that sometimes, `elem` could be slower than the explicit pattern matching. I remember there were some benchmarks on one particular case, that showed how moving to pattern matching on each case separately drastically decrease time 🐎